.eleventy.js

```diff
@@ -0,0 +1,22 @@
+module.exports = function(config) {
+  // A useful way to reference the context we are runing eleventy in
+  let env = process.env.ELEVENTY_ENV;
+
+  // Layout aliases can make templates more portable
+  config.addLayoutAlias('default', 'layouts/base.njk');
+
+  // pass some assets right through
+  config.addPassthroughCopy("./src");
```

Comment:
At some point, we may want to put the assets in a sub-folder of src so we don't automatically assume we should copy-over all the files in `src`